c/check.py

```diff
@@ -8,6 +8,8 @@
 import os
 import re
 import sys
+from multiprocessing import Pool
+from functools import partial
```

Comment:
My personal preference is to not import single functions (only modules or classes) because name clashes are too easy.

Reply:
Sure, I've changed it.